sycl/doc/extensions/SPIRV/SPV_INTEL_float_controls2.asciidoc

```diff
@@ -0,0 +1,140 @@
+SPV_INTEL_float_controls2
+=========================
+
+Name Strings
+------------
+
+SPV_INTEL_float_controls2
+
+Contact
+-------
+
+To report problems or to provide input on this extension, please open a new issue at:
+https://github.com/intel/llvm/issues
+
+Contributors
+------------
+
+- Gang Chen, Intel
+- Mariusz Merecki, Intel
+- Aleksander Us, Intel
+- Konstantin Vladimirov, Intel
+
+
+
+Notice
+------
+
+Copyright (c) 2020 Intel Corporation. All rights reserved.
+
+Status
+------
+
+Working Draft
+
+This is a preview extension specification, intended to provide early access to a feature for review and community feedback. When the feature matures, this specification may be released as a formal extension.
+
+Because the interfaces defined by this specification are not final and are subject to change they are not intended to be used by shipping software products. If you are interested in using this feature in your software product, please let us know!
+
+
+Version
+-------
+
+[width="40%",cols="25,25"]
+|========================================
+| Last Modified Date | 2020-04-24
+| Revision           | 2
+|========================================
+
+Dependencies
+------------
+
+This extension is written against the SPIR-V Specification,
+Version 1.5, Revision 2, Unified
+
+This extension requires SPIR-V 1.0.
+
+Overview
+--------
+
+This extension provides new execution modes to control floating-point computations by overriding an implementation’s default behavior for rounding modes and floating point modes.
```

Comment:
Could you clarify what the "floating point mode" is?

Reply:
I'm struggling to find a good wording for describing floating point mode.
Would you be able to help me here and propose some wording?

Reply:
Updated to the proposed test. Thank you Alexey.